cmd/sebak/cmd/genesis.go

```diff
@@ -136,7 +136,14 @@ func MakeGenesisBlock(addressStr, networkID, balanceStr, storageUri string, log
 		return "<public key>", fmt.Errorf("failed to create genesis block: %v", err)
 	}
 
-	log.Info("GenesisBlock created", "height", b.Height, "block", b)
+	log.Debug("GenesisBlock created", "block", b)
+	log.Info("GenesisBlock created",
```

Comment:
With just `log.Info`, the same log message will be printed, so I think `log.Debug` is redundant :)